vue/components/ui/generic-field-platforme/generic-field-platforme.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+    <div class="generic-field-container">
+        <img class="icon" v-bind:src="iconSrc" v-if="iconSrc" v-show="iconVisible" />
+        <input
+            class="generic-field-platforme"
+            v-bind:class="[iconVisible ? '' : 'icon-invisible', disabled ? 'disabled' : '']"
+            v-bind:type="type"
+            v-bind:value="value"
+            v-bind:disabled="disabled"
+            v-bind:placeholder="placeholder"
+            v-bind:required="required"
+            v-bind:autofocus="autofocus"
+            v-bind:style="'width: ' + width + 'px; ' + 'height: ' + height + 'px;'"
+            ref="input"
+            v-on:input="$emit('update:value', $event.target.value)"

```

Comment:
Use `updated`, it's enough

Reply:
Removed. It was only used for updating the suggestions. There are not suggestions in this component so it's not needed.